storm-core/pom.xml

```diff
@@ -42,6 +42,10 @@
             This is here as a work around to place it at the beginning of the classpath
             even though maven does not officially support ordering of the classpath.-->
         <dependency>
+            <groupId>uk.org.lidalia</groupId>
+            <artifactId>sysout-over-slf4j</artifactId>
+        </dependency>
```

Comment:
The new dependency seems OK, and it is unlikely that we will encounter some topology already using this, but could we please shade it anyways?